docs/cn/Configuring-Alluxio-with-S3.md

```diff
@@ -88,6 +88,10 @@ Alluxio提供了一个本地客户端与S3交互，默认情况下，当S3被配
 
 将`<USE_HTTPS>`设置为`true`或者`false`。如果设置为`true` (使用HTTPS)，同时需要设置`<HTTPS_PORT>`为服务提供商给定的HTTPS port，并且删除`alluxio.underfs.s3.endpoint.http.port`参数。 如果你将`<USE_HTTPS>`设置为`false`（即使用HTTP），同时也需要设置`<HTTPS_PORT>`为服务提供商给定的HTTPS port，并且删除`alluxio.underfs.s3.endpoint.https.port`参数。如果HTTP或HTTPS的port值没有设定，那么`<HTTP_PORT>`采用的默认端口为80，`<HTTPS_PORT>`采用的默认端口为443.
 
+### 使用v2的S3签名
```

Comment:
this is unrelated to the PR title ... please submit your change in a separate PR